recipes/ecell/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# install using pip from the whl files on PyPI
+
+if [ `uname` == Darwin ]; then
+    if [ "$PY_VER" == "2.7" ]; then
+        WHL_FILE=https://files.pythonhosted.org/packages/c3/fa/c1cf3e3e34b4531c57fe1de48e6537d02dc2751e29772a5533160a475dd7/ecell-4.1.4-cp27-cp27m-macosx_10_13_x86_64.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl
```

Comment:
Can we build from source instead?  We prefer not to just repackage wheels.  That way we can ensure that things are ABI compatible.

Reply:
@mariusvniekerk 
It may not be impossible, but it is difficult.
We created our whls with
https://github.com/ecell/ecell4/blob/master/appveyor.yml (for Windows)
https://github.com/ecell/ecell4-wheels/blob/master/.travis.yml (for Mac)
https://github.com/ecell/ecell4/blob/master/circle.yml (for manylinux)

Do you think these build scripts will work in conda-forge's build environment?

Reply:
yeah, should be fine,  conda-forge builds scary things all the time :D.  
You don't need to do the relatively gross wheel pieces luckily.  I'd recommend you make a build.sh and a bld.bat to cover the two main classes.